README.md

```diff
@@ -28,6 +28,23 @@ if they are valid then the cookie is updated to indicate this otherwise an error
 * if there is a cookie but it indicated the the authentication is expired then the user is sent off to Google to renew their session.
 On returning from Google the existing cookie is updated with the new expiry time.
 
+## Authorising to the s3 bucket
+If the application is running in the same AWS account as the pan-domain bucket (Workflow) then apps can access this by granting read
+access to the pan-domain bucket in the cloudformation script.
+For applications outside the Workflow account there is an example sts:AssumeRole script. You need to add your AWS accountId in the 
+parameters and upload this to the Workflow account and get the roleArn.
+
+In your PanDomainAuthActions include this:
+
+```
+override def awsCredentialsProvider  = {
+  new AWSCredentialsProviderChain(new STSAssumeRoleSessionCredentialsProvider({{roleArn}}, "yourapp"),
+                                  new ProfileCredentialsProvider("workflow"))
```

Comment:
Yeah it would be nice to have as few dependencies as possible for local dev, agreed. It is up to the client how they implement the provider chain, and so you could chose instead to use keys to default to if you'd prefer. I think in general it makes sense for applications that need access to resources in other accounts to use the credentials of the profile its accessing, which is why I've suggested this approach. Happy to discuss though?